.gitignore

```diff
@@ -7,6 +7,7 @@
 /pkg/
 /spec/reports/
 /tmp/
+/vendor
```

Comment:
I just didn't want to change and force push another commit since @luis-ca is the author. I can't think of any reason this is necessary for this PR though. Ok with a second commit that removes it?